pallets/gear/src/benchmarking/code.rs

```diff
@@ -452,6 +454,36 @@ pub mod body {
         FuncBody::new(Vec::new(), instructions)
     }
 
+    pub fn write_access_all_pages_instrs(
+        mem_size: u32,
+        mut head: Vec<Instruction>,
+    ) -> Vec<Instruction> {
+        for page in (0..mem_size)
+            .map(WasmPageNumber)
+            .flat_map(|p| p.to_gear_pages_iter())
+        {
+            head.push(Instruction::I32Const(page.offset() as i32));
+            head.push(Instruction::I32Const(42));
+            head.push(Instruction::I32Store(2, 0));
+        }
+        head
+    }
+
+    pub fn read_access_all_pages_instrs(
+        mem_size: u32,
+        mut head: Vec<Instruction>,
+    ) -> Vec<Instruction> {
+        for page in (0..mem_size)
+            .map(WasmPageNumber)
+            .flat_map(|p| p.to_gear_pages_iter())
```

Comment:
why do we need access to all gear pages, if only wasm pages make sense for the bench, isn't it?

Reply:
No, each native page must be accessed at least